tests/lock_test.py

```diff
@@ -108,13 +108,16 @@ def set_up_nginx_dir(root_path):
     """
     # Get the root of the git repository
     repo_root = check_call('git rev-parse --show-toplevel'.split()).strip()
-    conf_script = os.path.join(
-        repo_root, 'certbot-nginx', 'tests', 'boulder-integration.conf.sh')
-    # Prepare self-signed certificates for Nginx
+    # We add manually nginx_config module, because certbot_integration_tests may not be installed.
```

Comment:
You're right that this isn't exactly elegant. How about we leave this file and `boulder-integration.conf.sh` alone for now and in a subsequent PR move this test into `certbot-ci`?

Most of the code in this file would no longer be needed if it was running inside `certbot-ci`. We could have one test in the Certbot tests and one in the Nginx tests.

As an aside, I wonder if we should make another set of tests in `certbot-ci` in the future which would be system tests that don't require an ACME server. This file, most of our test for `cli.py` and `main.py`, and some of the current integration tests fall into this category.

Reply:
Keeping files like `tests/integration/_common.sh` around to help with running a local Pebble/Boulder and configuring Certbot to use it so you can run tests manually isn't a bad idea. I think most of the core team never does this (the staging server and cheap domains and VPSes are great), but I think it's nice to have for people who don't have those options.

If we keep `tests/integration/_common.sh`, I think we should:

1. Rename it.
2. Keep some of the boulder/pebble start scripts around.

I'd also like to see `tests/integration/_common.sh` simplified and the boulder/pebble start scripts refactored to share as much logic with `certbot-ci` that makes sense, but that could be done in a later PR if you prefer.

Reply:
So as discussed, I will keep for now:
1) `common.sh`
2) Pebble and Boulder setup bash scripts
3) nginx config bash script

They will be removed in a future PR to be replaced by appropriate solutions in Certbot-CI.

Reply:
Done.